lib/Common/Core/Assertions.h

```diff
@@ -77,10 +77,12 @@ extern int IsInAssert;
 
 #ifdef DBG
 #define AssertOrFailFast(x)                 Assert(x)
+#define AssertOrFailFastHR(x, hr)           Assert(x)
 #define AssertOrFailFastMsg(x, msg)         AssertMsg(x, msg)
```

Comment:
do you need the HR part for this one as well?

Reply:
I think it's not needed in this case, it's more that the rethrow by failfast loses the context info.

Reply:
Might as well add it; we use AssertOrFailFastMsg a lot, and in addition to the change below, I see this change enabling folks to write FailFast-with-HR in the future.